geomesa-z3/src/main/scala/org/locationtech/geomesa/curve/LegacyYearXZ3SFC.scala

```diff
@@ -0,0 +1,43 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.curve
+
+import java.time.temporal.ChronoUnit
+
+/**
+ * XZ3SFC with a legacy, incorrect max time value of 52 weeks. The max value is kept the same to ensure that
+ * index keys and query ranges are consistent. Any dates that exceed the original max time will be dropped into
+ * the last time bin, potentially degrading results for the last day or two of the year.
+ *
+ * @param g resolution level of the curve - i.e. how many times the space will be recursively quartered
```

Comment:
It is not quartered!  It is split into 8ths!;)

Reply:
good point this was probably copied from the xz2 index